game-core/src/main/java/games/strategy/triplea/ResourceLocationTracker.java

```diff
@@ -10,42 +15,28 @@
  */
 @UtilityClass
 class ResourceLocationTracker {
+  private static final String REQUIRED_ASSET_FOLDER = "baseTiles/";
```

Comment:
The variable name is a bit odd as there are other required asset folders, yet the name of the folder is 'baseTiles'. Is this just an example required folder to indicate when we have found the map resources? Does the actual folder matter less so than it is just one of several that we would expect to find?

Reply:
Yes, precisely. `baseTiles` is necessary, but not sufficient, to specify assets but sufficient to identify the asset path

Reply:
@DanVanAtta would you prefer `EXAMPLE_REQUIRED_ASSET_FOLDER`?

Reply:
Renamed to `REQUIRED_ASSET_EXAMPLE_FOLDER` and added comment